apps/android/app/src/main/java/chat/simplex/app/views/chat/item/CIFileView.kt

```diff
@@ -71,6 +104,17 @@ fun CIFileView(
 
   fun fileAction() {
     if (file != null) {
+      if (file.isVoiceMessage() && file.loaded) {
```

Comment:
I think voice message chat item logic should live in a separate file, similar to CIImageView - CIVoiceMsgView.

Also I don't think that whether it's a voice message should be determined by checking extensions as `isVoiceMessage`does - I believe we don't do this for images.. The branching will be possible based on MsgContent type - and if the format is unplayable (the check for extension can be more downstream) chat item can indicate it.